resources/views/comments/index.blade.php

```diff
@@ -4,18 +4,43 @@
 --}}
 @extends('master')
 
+@php
+    $commentBundleJson = $commentBundle->toArray();
+
+    $links = [
+        [
+            'title' => osu_trans('comments.index.nav_title'),
+            'url' => route('comments.index'),
+        ],
+    ];
+
+    $userJson = $commentBundleJson['user'] ?? null;
+    if ($userJson !== null) {
+        $links[] = [
+            'title' => $userJson['username'],
+            'url' => route('users.show', ['user' => $userJson['id']]),
+        ];
+        $links[] = [
+            'title' => osu_trans('comments.index.nav_comments'),
+            'url' => route('comments.index', ['user_id' => $userJson['id']]),
+        ];
+    }
+@endphp
+
 @section('content')
-    <div class="js-react--comments-index osu-layout osu-layout--full"></div>
+    @include('layout._page_header_v4', ['params' => [
+        'links' => $links,
+        'linksBreadcrumb' => true,
+        'theme' => 'comments',
+    ]])
+    <div class="osu-page osu-page--comments">
+        <div class="js-react--comments-index u-contents"></div>
 
-    {{-- temporary pagination to be used by react component above --}}
-    <div class="hidden">
-        <div class="js-comments-pagination">
-            @include('objects._pagination_v2', ['object' => $commentPagination])
-        </div>
```

Comment:
I think this was more not to have a pager with no body before the component finishes mounting 🤔

Reply:
it was because it's located inside the react component's `osu-page` (before this change)